grazel-gradle-plugin/src/main/kotlin/com/grab/grazel/tasks/internal/FormatBazelBuildFileTask.kt

```diff
@@ -88,30 +93,43 @@ abstract class FormatBazelFileTask : DefaultTask() {
          */
         fun register(
             project: Project,
+            buildifierScriptProvider: Provider<RegularFile>,
             configureAction: Task.() -> Unit
         ): TaskProvider<out Task> {
             val rootProject = project.rootProject
             if (project == rootProject) {
                 // Format work space
-                val formatWorkspace = rootProject.register(FORMAT_WORK_SPACE_FILE_TASK) {
+                val formatWorkspace = rootProject.register(
+                    taskName = FORMAT_WORK_SPACE_FILE_TASK,
+                    buildifierScriptProvider = buildifierScriptProvider,
+                ) {
                     bazelFile = File(rootProject.projectDir, WORKSPACE)
                     description = "Format $WORKSPACE file"
 
                     configureAction(this)
                 }
                 // Format build.bazel
-                val formatBuildBazel = rootProject.register(FORMAT_BUILD_BAZEL_FILE_TASK) {
+                val formatBuildBazel = rootProject.register(
+                    taskName = FORMAT_BUILD_BAZEL_FILE_TASK,
+                    buildifierScriptProvider = buildifierScriptProvider,
+                ) {
                     description = "Format $BUILD_BAZEL file"
 
                     configureAction(this)
                 }
                 // Aggregating task to depend on above
-                return rootProject.register(FORMAT_BAZEL_FILE_TASK) {
+                return rootProject.register(
+                    taskName = FORMAT_BAZEL_FILE_TASK,
+                    buildifierScriptProvider = buildifierScriptProvider,
```

Comment:
This is just a aggregating task, `buildifierScriptProvider` need not be passed here.